solidity/test/token/TokenBridgeCctp.t.sol

```diff
@@ -1036,8 +1039,27 @@ contract TokenBridgeCctpV2Test is TokenBridgeCctpV1Test {
             amount
         );
 
-        deal(quotes[1].token, address(this), quotes[1].amount);
-        IERC20(quotes[1].token).approve(address(router), quotes[1].amount);
+        assertEq(quotes[1].token, usdc);
+        uint256 usdcQuote = quotes[1].amount;
+
+        deal(usdc, address(this), usdcQuote);
+        IERC20(usdc).approve(address(router), usdcQuote);
+
+        vm.expectEmit(true, true, true, true, address(router.tokenMessenger()));
```

Comment:
Can we test message reception? 

It'd be nice if we could confirm that the proper amount is received upon message delivery. I'd like to see an `expectEmit` on the [`mintAndWithdraw`](https://github.com/circlefin/evm-cctp-contracts/blob/5e893866d49326d62d3ecf3f7fb599dd71787469/src/v2/BaseTokenMessenger.sol#L358) event

Reply:
fdf642f50e77226335c1cb1e7fe368dd0b92e52b